src/konnektor/network_planners/generators/cyclic_network_generator.py

```diff
@@ -27,40 +27,45 @@ def __init__(
         _initial_edge_lister: NetworkGenerator = None,
     ):
         """
-        The `CyclicNetworkGenerator` is generating a network based on many network cycles. T
-        his is of interest for analyzing the uncertainty of FE Estimates along the thermodynamic cycles and possibly correct the estimates with cycle closure analysis.
+        A `NetworkGenerator` that generates a network based on many network cycles.
+        This is of interest for analyzing the uncertainty of FE Estimates along thermodynamic
+        cycles and possibly for correcting the estimates with cycle closure analysis.
 
         The greedy algorithm builds the network up from a nodewise perspective.
-        For each node, the algorithm generates all cycles of size `cycle_sizes` and assigns a score to each cylce as the sum of all sub-scores.
-        Next it selects the `node_present_in_cyces` best score perfoming and node diversity increasing (see below) cycles per node.
+        For each node, the algorithm generates all cycles of size `cycle_sizes` and assigns a score to each cycle as the sum of all sub-scores.
+        Next, it selects the `node_present_in_cycles` best score performing and node diversity increasing (see below) cycles per node.
         The set of selected `Transformations` constructs the graph.
-        The node diversity criterion is an addition, which biases to spread the cycles on the graph eaqually between all `Components`
+        The node diversity criterion is an addition which biases to spread the cycles on the graph equally between all `Components`.
 
-        The number of cylces, around each `Component` can be defined by `component_present_in_cycles` and allowed cylce size can be tweaked with `cycle_sizes`. For `cycle_sizes` either an integer for providing an expected cycle size (e.g. `3`) or a range of allowed cycle sizes (e.g. `[3,4]`).
+        The number of cycles around each `Component` can be defined by `component_present_in_cycles`
+        and the allowed cycle size can be tweaked with `cycle_sizes`.
 
-        This layout has a well distributed connectivity between all `Component` s which increases the robustness very well, but still allows for a better graph score then the Twin Star Network, as the connectivity distribution is biased not enforced.
-        The large number of cycles might be very useful for statical analysis.  Nevertheless, the network has an increased amount of `Transformation` s
+        This layout has well-distributed connectivity between all `Component` s, which increases the robustness very well,
+        but still allows for a better graph score then the Twin Star Network, as the connectivity distribution is biased and not enforced.
+        The large number of cycles might be very useful for statistical analysis.  Nevertheless, the network has an increased amount of `Transformation`s.
```

Comment:
I think these `s might be the reason why docs are not building right now?
```suggestion
        The large number of cycles might be very useful for statistical analysis.  Nevertheless, the network has an increased amount of `Transformation` s.
```